### ISSUE TITLE
Pre Sale Question - Road Map :)

### ISSUE THREAD

**Issue by bentalgad** [dudaster/ele-custom-skin#47]
I see you sell a pro version now and you write "Future!!! Many more awesome things." can you say if you have general ideas on what that might include? what directions / subjects / areas are other futures might be... :)

Thanks! Good Luck!

**Comment by bentalgad** [comment]
And more specifically - do you plan adding something that has to do with ACF repeaters?

**Comment by dudaster** [comment]
Yes I am. But it will be a different plugin and it will be free.

**Comment by bentalgad** [comment]
Wow great. waiting to see that. and about any road map to this plugin?

**Comment by dudaster** [comment]
This week I'll release dynamic keyword feature in ECS PRO where you can use any widget you want and add a keyword like {{post_title}} it would be replaced with the specific value or any other  {{custom_field_slug}} not limited to ACF, Pods, Toolset etc.

So hopefully next month I'll release it.

**Comment by bentalgad** [comment]
That's Nice. for now i use the build-in ACF Shortcode `[acf field="field_name"]` but of course it doesn't work in every place. i understand your keyword will work any where you put it right? even in the middle of the content for example (like beaver does)... do you have more future ideas ECS PRO?

**Comment by sigersmit** [comment]
I also bought the Pro version and I would also like to see what kind of awesome things are coming up :-) Also a little demo on how the dynamic keywords work and specific use cases, because that's not clear to me.. Thanks!

**Comment by bentalgad** [comment]
Still waiting for the ACF Repeater :) there are some plugins that do that by now, but none of them is really stable and nice. i'm really curious to see how you go about it.

**Comment by dalguji** [comment]
I am still waiting for the ACF Repeater too!
